drivers/dac/Kconfig.mcp4725

```diff
@@ -0,0 +1,15 @@
+# DAC configuration options
+
+# Copyright (c) 2021 Laird Connectivity
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config DAC_MCP4725
+	bool "Microchip MCP4725 DAC driver"
+	depends on I2C
+	help
+	  Enable the driver for the Microchip MCP4725.
+
+if DAC_MCP4725
+
```

Comment:
seems like you can remove this if/endif block as its empty.

Reply:
Thank you, this is now fixed.